packages/analytics/analytics-chart/src/components/TopNTable.vue

```diff
@@ -230,7 +230,8 @@ const getValue = (record: AnalyticsExploreRecord): string => {
 </script>
 
 <style lang="scss" scoped>
-@import '../styles/base';
+@use "../styles/base" as *;
```

Comment:
I don’t think you need to add the namespace if these aren’t variables (same comment for all instances)

Reply:
(I couldn’t pull up the file on mobile for some reason so maybe these aren’t vars)

Reply:
Originally there's only `@import "./mixins.scss";` in it, I added variable inclusion into it as well. There's no CSS output in it.